packages/fabric8-ui/src/app/dashboard-widgets/space-settings-widget/space-settings-widget.component.html

```diff
@@ -0,0 +1,130 @@
+<f8-feature-toggle
+  featureName="Analyze.WorkItemOverviewCard"
```

Comment:
Isn't this space settings widget should have there own feature flag label?

Reply:
Fixed it with a new feature flag